static/app/gettingStartedDocs/python/celery.tsx

```diff
@@ -60,15 +86,88 @@ ${sentryInitContent}
     ],
     additionalInfo: (
       <Fragment>
-        {t(
-          'Additionally, the Sentry Python SDK will set the transaction on the event to the task name, and it will improve the grouping for global Celery errors such as timeouts.'
-        )}
+        <h5>{t('Standalone Setup')}</h5>
+        {t("If you're using Celery standalone, there are two ways to set this up:")}
+        <ul>
+          <li>
+            {tct(
+              "Initializing the SDK in the configuration file loaded with Celery's [code:--config] parameter",
+              {
+                code: <code />,
+              }
+            )}
+          </li>
+          <li>
+            {tct(
+              'Initializing the SDK by hooking it to either the [celerydInit: celeryd_init] or [workerInit: worker_init] signals:',
+              {
+                celerydInit: (
+                  <ExternalLink href="https://docs.celeryq.dev/en/stable/userguide/signals.html?#celeryd-init" />
+                ),
+                workerInit: (
+                  <ExternalLink href="https://docs.celeryq.dev/en/stable/userguide/signals.html?#worker-init" />
+                ),
+              }
+            )}
+            <CodeSnippet dark language="python">
+              {t(`import sentry_sdk
+from celery import Celery, signals
+
+app = Celery("myapp")
+
+#@signals.worker_init.connect
+@signals.celeryd_init.connect
+def init_sentry(**_kwargs):
+    sentry_sdk.init(...)  # same as above
+              `)}
```

Comment:
`t()` is for translating strings. Guess that is not necessary here 🙂 
```suggestion
              {`import sentry_sdk
from celery import Celery, signals

app = Celery("myapp")

#@signals.worker_init.connect
@signals.celeryd_init.connect
def init_sentry(**_kwargs):
    sentry_sdk.init(...)  # same as above
              `}
```